.storybook/assets/css/preview.css

```diff
@@ -1,18 +1,24 @@
 :focus,
 :focus-visible {
-  outline: none;
+  outline: transparent;
 }
 
-a {
+:global(a) {
```

Comment:
linter was unhappy with the un-scoped styles here (a rule which we rightly added to prevent components styles from "leaking")